### DISCUSSION TITLE
Is it possible to change `*`?

### DISCUSSION THREAD

**Discussion by semi710** [jesseduffield/lazygit#3400] [Q&A] [answered]
is it possible to change `*` in branch preview to some kind of nerd font icon through config?
and the `|` symbol also? 
as then this can create a nice looking preview in cli also 
`*` -> ``
`|` -> `│`
just in config to replace them?

**Comment by stefanhaller** [comment, marked as answer]
Are you looking for something like this?
```yaml
git:
  branchLogCmd: sh -c 'git log --graph --color=always --abbrev-commit --decorate --oneline {{branchName}} -- | sed -e s/\\*/●/ -e s/\\|/│/'
```

**Reply by semi710** [reply]
exactly what i was looking for ........ 
Thanks ig i have quite less experience in git to think of this on my head.

Not to aanoy is it possible to replace the `M` `D` `?` symbols also with some nerd fonts?
as in my neovim i use 
```
│ in green to present addition
│ in red for deletion
```

in general if lazygit provides these icon replacement it would be nice.
